### DISCUSSION TITLE
support turbopack bundler

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/nuxt#16523] [Ideas]
This is a placeholder issue to track future support for turbopack. Right now turbopack only has support for React and no way to be extended via plugins. They say:

> As of today, Turbopack can be used in Next.js v13. In the future we will be releasing a standalone CLI, plugin API, and support for other frameworks such as Svelte and Vue.

If and when it becomes possible to write plugins for turbopack or use it with vue, we can explore whether to add this as a third experimental bundler option to Nuxt (the capability is already there to use an alternative bundler without any changes to Nuxt itself). It may be that vite would also consider using turbopack under the hood.

Either way, now is too early to tell other than to say we're tracking the issue with interest.

https://turbo.build/pack/docs

**Comment by davatron5000** [comment]
Dumb question, but would this be an easy pathway to make Nuxt 2 faster?

**Comment by wesbragagt** [comment]
I would love to see that happen for Nuxt2 and beyond

**Comment by dev-adewale** [comment]
I would advice to wait till Vite starts using Turbopack under the hood, Nuxt shouldn't port to Turbopack for bundling. Keep it in the ecosystem, what do y'all think? 💭

**Comment by podlebar** [comment]
i would advice to wait and see if Turbopack really can keep what it promises.. until now it's pure marketing without proper or real benchmarks.

**Comment by dev-adewale** [comment]
> i would advice to wait and see if Turbopack really can keep what it promises.. until now it's pure marketing without proper or real benchmarks.

I agree with you on that, even Evan You perform a benchmark test on it today. When compared to Vite, Turbopack is even slower
